packages/react-sdk/src/useObservable.ts

```diff
@@ -2,31 +2,45 @@
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import { type Observable, type PersistedModel } from '@river-build/sdk'
 
-export type ObservableConfig<T> = {
-    fireImmediately?: boolean
-    onUpdate?: (data: T) => void
-    onError?: (error: Error) => void
-    onSaved?: (data: T) => void
-}
-
-type ObservableReturn<T> = {
-    data: T | undefined
```

Comment:
Is there any case where the first data will be an undefined one and not of the type T? cc @texuf